### ISSUE TITLE
PHP library returns wrong image path for toImage

### ISSUE THREAD

**Issue by rovansteen** [joypixels/emoji-toolkit#22]
Both `toImage` and `shortnameToImage` return a path that returns an S3 AccessDenied error.

```php
(new \JoyPixels\JoyPixels())->getClient()->toImage(':smile:')
```
Returns:
```
<img class="joypixels" alt="&#x1f604;" title=":smile:" src="https://cdn.jsdelivr.net/joypixels/assets/5.5/png/32/1f604.png"/>
``` 
https://cdn.jsdelivr.net/joypixels/assets/5.5/png/32/1f604.png

I'm using the latest version (`5.5.0`)

**Comment by caseyahenson** [comment]
This has been corrected in the just-released 5.5.1, with the missing url parameter `/unicode` now included. Thank you, and let me know how it's working for you.

**Comment by rovansteen** [comment]
Cheers, seems to be working fine!
